projects/plugins/jetpack/extensions/blocks/dialogue/attributes.js

```diff
@@ -1,15 +1,21 @@
 export default {
-	participantSlug: {
-		type: 'string',
+	participant: {
+		type: 'object',
 	},
 	timestamp: {
 		type: 'string',
 		default: '00:00',
 	},
+	showTimestamp: {
+		type: 'boolean',
+		default: false,
+	},
```

Comment:
Add the `showTimestamp` setting to each dialogue block, so we have it available when rendering in the `save` function.

Reply:
Also, we can show or hide the timestamp per Dialogue, something that as noted in a feedback